gcloud/datastore/helpers.py

```diff
@@ -218,44 +218,6 @@ def _get_value_from_property_pb(property_pb):
     return _get_value_from_value_pb(property_pb.value)
 
 
-def _set_protobuf_property(property_pb, name, value, indexed):
```

Comment:
Rather than resurrect `_set_protobuf_property`, de03993 folds in the change (skipping properties with empty-list values) into `_assign_entity_to_mutation`.